test.sh

```diff
@@ -0,0 +1,9 @@
+set -ex;
+root=`pwd`
+yes | lamdera reset || true
```

Comment:
This `|| true` is a smell, I'll come back and remove it once I've done a release with the fix that led to this (edge case where reset fails if the folder it's trying to purge doesn't exist).